packages/amplify-cli-npm/index.ts

```diff
@@ -16,4 +16,4 @@ export const install = async (): Promise<void> => {
   return binary.install();
 };
 
-// force version bump to 12.5.0
```

Comment:
if you merge this as is version will be `12.6` given `feat` in the PR description